src/py/flwr/cli/new/new.py

```diff
@@ -173,9 +165,88 @@ def new(
+            f"{import_name}/models.py": {"template": "app/code/flwrtune/models.py.tpl"},
+            f"{import_name}/dataset.py": {
+                "template": "app/code/flwrtune/dataset.py.tpl"
+            },
+            f"{import_name}/conf/config.yaml": {
+                "template": "app/code/flwrtune/config.yaml.tpl"
+            },
+            f"{import_name}/conf/static_config.yaml": {
+                "template": "app/code/flwrtune/static_config.yaml.tpl"
+            },
+        }
+
+        # Task specific context
+        context["llm_task_str"] = llm_task_str
+        context["fraction_fit"] = "0.2" if llm_task_str == "code" else "0.1"
+        context["task_name"] = (
+            "General NLP"
+            if llm_task_str == "generalnlp"
+            else (
+                "Finance"
+                if llm_task_str == "finance"
+                else "Medical" if llm_task_str == "medical" else "Code"
+            )
+        )
+        context["num_clients"] = (
+            "20"
+            if llm_task_str in ("generalnlp", "medical")
+            else "50" if llm_task_str == "finance" else "10"
+        )
+        context["dataset_name"] = (
+            "vicgalle/alpaca-gpt4"
+            if llm_task_str == "generalnlp"
+            else (
+                "FinGPT/fingpt-sentiment-train"
+                if llm_task_str == "finance"
+                else (
+                    "medalpaca/medical_meadow_medical_flashcards"
+                    if llm_task_str == "medical"
+                    else "lucasmccabe-lmi/CodeAlpaca-20k"
+                )
+            )
+        )
+    else:
+        files = {
+            ".gitignore": {"template": "app/.gitignore.tpl"},
+            "README.md": {"template": "app/README.md.tpl"},
+            "pyproject.toml": {"template": f"app/pyproject.{framework_str}.toml.tpl"},
+            f"{import_name}/__init__.py": {"template": "app/code/__init__.py.tpl"},
+            f"{import_name}/server.py": {
+                "template": f"app/code/server.{framework_str}.py.tpl"
+            },
+            f"{import_name}/client.py": {
+                "template": f"app/code/client.{framework_str}.py.tpl"
+            },
+        }
+
+        # Depending on the framework, generate task.py file
+        frameworks_with_tasks = [
+            MlFramework.PYTORCH.value.lower(),
+            MlFramework.JAX.value.lower(),
+            MlFramework.HUGGINGFACE.value.lower(),
+            MlFramework.MLX.value.lower(),
+            MlFramework.TENSORFLOW.value.lower(),
+        ]
+        if framework_str in frameworks_with_tasks:
+            files[f"{import_name}/task.py"] = {
+                "template": f"app/code/task.{framework_str}.py.tpl"
+            }
+
     for file_path, value in files.items():
         render_and_create(
-            file_path=os.path.join(project_dir, file_path),
+            file_path=os.path.join(os.path.join(os.getcwd(), package_name), file_path),
```

Comment:
Cannot we leave this as it was before?

Reply:
It will get a `too many local variables` error from test